src/library-authoring/component-info/ComponentManagement.test.tsx

```diff
@@ -0,0 +1,18 @@
+import {
+  initializeMocks,
+  render,
+  screen,
+} from '../../testUtils';
+import { mockLibraryBlockMetadata } from '../data/api.mocks';
+import ComponentManagement from './ComponentManagement';
+
+describe('<ComponentManagement />', () => {
+  it('should render draft status', async () => {
+    initializeMocks();
+    mockLibraryBlockMetadata.applyMock();
+    render(<ComponentManagement usageKey={mockLibraryBlockMetadata.usageKeyNeverPublished} />);
+    expect(await screen.findByText('Draft')).toBeInTheDocument();
+    expect(screen.queryByText('Draft(Never Published)')).not.toBeInTheDocument();
+    expect(screen.queryByText('Draft saved on June 20, 2024 at 13:54 UTC.)')).not.toBeInTheDocument();
+  });
```

Comment:
Could you add a test for the component published state too?

Reply:
Done: https://github.com/openedx/frontend-app-course-authoring/pull/1275/commits/98ebc92ca70d3317f5a9f7ca3b50c48bb66c16f8